src/test/java/com/example/amazoncom/BrowserStackHomePage.java

```diff
@@ -0,0 +1,40 @@
+package com.example.amazoncom;
+
+import static org.testng.Assert.assertEquals;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class BrowserStackHomePage {
```

Comment:
this is a page object for the Amazon home page, Why do you use BrowserStack naming? It's okay to look into BrowserStack examples, but you need to use your own naming when you do "copy-paste" of a solution